.gitignore

```diff
@@ -3,4 +3,4 @@ build
 dist
 .idea/
 *.egg-info
-/.tox
```

Comment:
Why do you remove the empty line at the end of the file?

Reply:
Must have been when I deleted the venv/ line.
Sorry. I will add it again.